proto/ibc/localhost/localhost.proto

```diff
@@ -24,5 +25,8 @@ message ClientState {
     (gogoproto.moretags) = "yaml:\"chain_id\""
   ];
   // self latest block height
-  uint64 height = 3;
+  ibc.client.Height height = 3 [
+    (gogoproto.moretags) = "yaml:\"height\"",
+    (gogoproto.nullable) = false
```

Comment:
does this need to be a pointer for proto?

Reply:
Would this be a problem for proto encoding/decoding? Given the choice, i prefer it to not be a pointer since there's no reason to have a nil Height for `ClientState`/`ConsensusState` and not having nil checks everywhere is simpler